samples/metrics/tracing/sample.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Sending metrics data to trace log
 component: Metrics
-reviewed: 2017-04-04
+versions: '[1.0,1.1)'
```

Comment:
in a doco page yes. in a sample u would just use the new api, no need for the versions header